app/src/main/res/values/strings.xml

```diff
@@ -964,7 +962,19 @@
     <string name="label_shared_file_too_large">You can only share a file up to %d MB.</string>
     <string name="label_file_saved_to_device">The file was saved to your device.</string>
 
-    <!-- Expirable messages -->
+    <!-- Expiring messages -->
+    <string name="self_deleting_message_time_left">Self-deleting message • %1$s</string>
+    <string name="self_deleting_message_label">Self-deleting message</string>
+    <string name="self_deleting_messages_team_setting_disabled">Self-deleting messages are disabled.</string>
+    <string name="self_deleting_messages_team_setting_enabled">Self-deleting messages are enabled. You can set a timer before writing a message.</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout">Self-deleting messages are now mandatory. New messages will self-delete after %1$s.</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout_seconds">10 seconds</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout_minutes">5 minutes</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout_one_hour">1 hour</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout_one_day">1 day</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout_one_week">1 week</string>
+    <string name="self_deleting_messages_team_setting_enabled_enforced_timeout_four_weeks">4 weeks</string>
```

Comment:
there is this 
![image](https://user-images.githubusercontent.com/15973138/234803851-ef22e618-1cd3-4bca-b9ca-911f5051648f.png)

no need to add this